.github/workflows/workflow.yml

```diff
@@ -46,16 +46,15 @@ jobs:
   composer:
     name: Composer
     needs: [check]
-    if: |
-      needs.check.outputs.composer > 0 ||
-      needs.check.outputs.workflow > 0
+#    if: |
+#      needs.check.outputs.composer > 0 ||
+#      needs.check.outputs.workflow > 0
```

Comment:
could we remove these lines instead or we need them in the future?

Reply:
I'd like to keep them for now. I already have an idea to fix it, but i need some time for tests.